configMaps/dep_patch.yml.mustache

```diff
@@ -0,0 +1,20 @@
+spec:
+  template:
+    spec:
+      serviceAccountName: {{namespace}}
+      initContainers:
+        - name: copy-default-config
+          volumeMounts:
+            - mountPath: /var/jenkins_jobs/jobs/{{repo}}
+              name: {{volume}}
+      volumes:
+      - name: csc-shouv-jenkins-job-test-jenkins-volume
```

Comment:
`shouv`?

Reply:
Restructured the entire config map template to get rid of similar issues